app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -130,6 +130,15 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, "/repositories/:id/export", N_("Export a repository")
+    param :id, :identifier, :required => true, :desc => N_("repository ID")
+    param :since, String, :desc => N_("ISO 8601 date of last export (ex: 2010-01-01T12:00:00), useful for exporting deltas.")
```

Comment:
In other areas of the application, we use a different date time format, see [1] [2] for some examples (there are more I'm sure). Should we keep this consistent? Or simply support any datetime format that Rails can convert? 


[1] https://github.com/Katello/katello/blob/068062449b30e985695aa6071837290e4afbd3df/test/lib/validators/content_view_erratum_filter_rule_validator_test.rb#L29
[2] https://github.com/Katello/katello/blob/068062449b30e985695aa6071837290e4afbd3df/engines/bastion_katello/test/sync-plans/new-sync-plan.controller.test.js#L37